### ISSUE TITLE
Vscode error

### ISSUE THREAD

**Issue by DrYSG** [badochov/ocamlformatter-vscode#1]
Just installed this extension and  the latest ocamlformat via opam (windows 10).
I am getting this error:

``
Error: Command failed: dirname d:\CRIME\ocommander\src\ocommander.ml 'dirname' is not recognized as an internal or external command, operable program or batch file.
```
the file d:\CRIME\ocommander\src\ocommander.ml exists and that is what I am trying to format. No idea what dirname is and how to set it.

**Comment by badochov** [comment]
Hi, I know about this issue, but unortunatelly I'm not sure how to fix it.

**Comment by badochov** [comment]
Are you using WSL in vscode? Because it seems as if you use normal windows console on which you can't even have ocaml installed

**Comment by smorimoto** [comment]
Does it happen with the official extension, the OCaml Platform?
https://marketplace.visualstudio.com/items?itemName=ocamllabs.ocaml-platform

**Comment by OscarLahaie** [comment]
I have the same issue and my ocamlplatform formatter is not working anymore too

**Comment by badochov** [comment]
Fixed in version 2.0.0
